slave/configs.py

```diff
@@ -187,6 +187,18 @@ def __init__(self, engine, shell):
         else:
             self.omit_ = True
 
+class WebRender(Default):
+    def __init__(self, engine, shell):
+        super(WebRender, self).__init__(engine, shell)
+        if engine == "firefox" and not shell:
+            self.prefs_["gfx.webrender.enabled"] = True
+            self.prefs_["gfx.webrender.blob-images"] = True
+            self.prefs_["image.mem.shared"] = True
+            # This last one is for Linux, and a no-op on other platforms.
+            self.prefs_["layers.acceleration.force-enabled"] = True
+        else:
+            self.omit_ = True
+
```

Comment:
We don't need to inherit from E10S or another class to pick up the other preferences related to e10s or other features?

Reply:
Good question! I don't think so. The E10S config class could even be removed because it's now the default on Firefox (ditto for Stylo, I think?). In both cases, this probably means touching the database control tasks, bleh.